ghcide/src/Development/IDE/LSP/LanguageServer.hs

```diff
@@ -126,10 +126,9 @@ runLanguageServer options inH outH getHieDbLoc defaultConfig onConfigurationChan
             registerIdeConfiguration (shakeExtras ide) initConfig
 
 
-            let handleServerException (Left e) = do
+            let handleServerException (Left e) =
                     logError (ideLogger ide) $
                         T.pack $ "Fatal error in server thread: " <> show e
-                    exitClientMsg
```

Comment:
Did you really want to remove this and why?

Reply:
nvm, I'm not going to remove that, just for resolving the merge conflicts online to run CI